engine.py

```diff
@@ -254,7 +254,7 @@ def init_engine(self):
         if maya_ver.startswith("Maya "):
             maya_ver = maya_ver[5:]
         if maya_ver.startswith(("2014", "2015", "2016", "2017")):
-            self.log_debug("Running Maya version %s" % maya_ver)
+            self.logger.debug("Running Maya version %s" % maya_ver)
```

Comment:
The thought of changing it did cross my mind. I think i saw it in one of the previous PRs you or @thebeeland alerted me to. But i did not think of the benefits of doing so. I will make those changes, I think you give strong arguments to do it now. Hopefully I don't break anything in the process :D